### ISSUE TITLE
Conflict with s1n7ax/nvim-window-picker related to here or not ? 

### ISSUE THREAD

**Issue by DrummyFloyd** [sphamba/smear-cursor.nvim#38]
hi , i finnaly found the culpit haha 

seems that your plugins do some conflicting stuff with `s1n7ax/nvim-window-picker`
once i activate `smear`  i got some `nil` values upon `windows picker`

tbh, dunno where i should put the issue , so i'll try first here coz you are the new comer ^^

```log
                     E5108: Error executing lua: ...ker/lua/window-picker/hints/floating-big-letter-hint.lua:130: attempt to index local 'char' (a nil value)
stack traceback:
	...ker/lua/window-picker/hints/floating-big-letter-hint.lua:130: in function 'draw'
	...indow-picker/lua/window-picker/pickers/window-picker.lua:63: in function 'pick_window'
	...y/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:854: in function 'open_file'
	...y/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:731: in function 'open'
	...y/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:755: in function 'open_with_cmd'
	...y/neo-tree.nvim/lua/neo-tree/sources/common/commands.lua:870: in function 'open_with_window_picker'
	...o-tree.nvim/lua/neo-tree/sources/filesystem/commands.lua:209: in function <...o-tree.nvim/lua/neo-tree/sources/filesystem/commands.lua:208>
```

## Environment
OK config 
```lua
vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system "curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua")()

require("lazy.minit").repro({
    spec = {
        { "LazyVim/LazyVim", import = "lazyvim.plugins" },
        -- add any other plugins here
        {
            "nvim-neo-tree/neo-tree.nvim",
            dependencies = {
                "s1n7ax/nvim-window-picker",
                opts = {
                    hint = "floating-big-letter",
                    show_prompt = false,
                    filter_rules = {
                        autoselect_one = true,
                    },
                },
            },
        },
    },
})

```

NOT OK with `smear`

```lua
vim.env.LAZY_STDPATH = ".repro"
load(vim.fn.system "curl -s https://raw.githubusercontent.com/folke/lazy.nvim/main/bootstrap.lua")()

require("lazy.minit").repro({
    spec = {
        { "LazyVim/LazyVim", import = "lazyvim.plugins" },
        { import = "lazyvim.plugins.extras.ui.smear-cursor" },
        -- add any other plugins here
        -- tested with vanilla integration too
        -- {
         --   "sphamba/smear-cursor.nvim",
        -- },

        {
            "nvim-neo-tree/neo-tree.nvim",
            dependencies = {
                "s1n7ax/nvim-window-picker",
                opts = {
                    hint = "floating-big-letter",
                    show_prompt = false,
                    filter_rules = {
                        autoselect_one = true,
                    },
                },
            },
        },
    },
})

```
- OS:
![image](https://github.com/user-attachments/assets/129cd2f4-1556-47ef-9d24-5bb6fabbb4e9)

- Terminal:
❯ kitty --version
kitty 0.37.0 created by Kovid Goyal

- Neovim version:
❯ nvim --version
NVIM v0.10.2
Build type: Release
LuaJIT 2.1.1713484068
Run "nvim -V1 -v" for more info

- Smear-cursor version/commit:
main:29a2c0e

**Comment by sphamba** [comment]
Fixed by #45 

I added a way to filter out the floating windows created by smear-cursor. You can add this to the window-picker config:
```lua
require("window-picker").setup({
  filter_rules = {
    bo = {
      filetype = { "smear-cursor" }
    }
  }
})
```

**Comment by DrummyFloyd** [comment]
thank you !!!
